x-pack/plugins/enterprise_search/public/applications/workplace_search/views/overview/onboarding_steps.tsx

```diff
@@ -58,16 +59,18 @@ const ONBOARDING_USERS_CARD_DESCRIPTION = i18n.translate(
 );
 
 export const OnboardingSteps: React.FC = () => {
+  const {
+    isFederatedAuth,
+    organization: { name, defaultOrgName },
+    account: { isCurated, canCreateInvitations },
```

Comment:
I know this is a tech update PR, so it's not a blocker in any way.
@scottybollinger Do you know if we still need these values or are they legacy and we can remove them?

Reply:
I don't know for sure but I am taking a note to find out. Thanks!